executor/platform_compat/windows/signals.py

```
@@ -0,0 +1,134 @@
+        """
+        try:
+            # Try to send CTRL_BREAK_EVENT first
+            # This works for console applications
+            kernel32 = ctypes.windll.kernel32
+            result = kernel32.GenerateConsoleCtrlEvent(
+                win32con.CTRL_BREAK_EVENT if self._available else 1, pid
+            )
+            if result:
+                return True
+        except Exception:
+            pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)